sound/soc/intel/common/soc-acpi-intel-cht-match.c

```diff
@@ -174,6 +174,13 @@ struct snd_soc_acpi_mach  snd_soc_acpi_intel_cherrytrail_machines[] = {
 		.sof_fw_filename = "sof-cht.ri",
 		.sof_tplg_filename = "sof-cht-cx2072x.tplg",
 	},
+	{
+		.id = "104C5122",
+		.drv_name = "sof_pcm512x",
+		.sof_fw_filename = "sof-cht.ri",
+		.sof_tplg_filename = "sof-cht-src-50khz-pcm512x.tplg",
+	},
+
```

Comment:
can you split this in another patch, that would help me submit upstream? Thanks!

Reply:
@plbossart you mean split patch 2 of the series into 2 parts - separately for soc-acpi-intel-bxt-match.c and soc-acpi-intel-cht-match.c? Ok, will do

Reply:
yes, thank you